### ISSUE TITLE
No sound when casting to Chromecast

### ISSUE THREAD

**Issue by gpzetje** [segler-alex/RadioDroid#829]
When I use the cast button the app connects to Chromecast. A RadioDoid drawing appears on TV, but sound still comes from my phone.
Casting works flawlessly with other apps (like Netflix).
Where do I begin looking for a solution?

**Comment by gpzetje** [comment]
<solved>
Changing channels opened a extra menu with MPD option, it also showed a Chromecast option.
Selected that and changed back to the stream I wanted hear in the first place.

**Comment by werman** [comment]
Yeah, the icon needs a change, since now it's everything and not only mpd.
